From 69119ca8ee05628d13577301594d268c624556a1 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Tue, 24 Jun 2025 09:11:10 +0200
Subject: [PATCH] Bump libpisp to 1.2.1

---
 subprojects/libpisp.wrap | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subprojects/libpisp.wrap b/subprojects/libpisp.wrap
index 8b62c036f..0e0c7baaf 100644
--- a/subprojects/libpisp.wrap
+++ b/subprojects/libpisp.wrap
@@ -2,5 +2,5 @@
 
 [wrap-git]
 url = https://github.com/raspberrypi/libpisp.git
-revision = v1.2.0
+revision = v1.2.1
 depth = 1
